engine/src/exec-interface-graphic.cpp

```diff
@@ -724,7 +767,14 @@ void MCGraphic::SetMarkerPoints(MCExecContext& ctxt, uindex_t p_count, MCPoint*
     }
     
     if (flags & F_MARKER_DRAWN && flags & F_MARKER_OPAQUE)
```

Comment:
If you can toggle markerOpaque flag (markerFilled prop maybe?) then that needs to be tested in the same way as the filled flag.